tests/data_reader.py

```diff
@@ -111,22 +120,31 @@ def read_chunk(self):
                 'data_flags': [data_flags],
                 'data': [data lines],
                 'errors': {errors},
+                'existing_schema': schema_string,
                 'schema': schema_string
             }
         Returns None if there are no more test chunks.
         """
-        data_flags, records = self.read_data_section()
+        data_flags, records, line = self.read_data_section()
```

Comment:
nit: same comment as below, maybe rename `line` to `lineno` to avoid confusion?

Reply:
This has been implemented in 411f402